### DISCUSSION TITLE
How to control the yaw Angle of fan in fast.farm?

### DISCUSSION THREAD

**Discussion by clun1** [OpenFAST/openfast#1558] [General]
How to control the yaw Angle of fan in fast.farm?

**Comment by jjonkman** [comment]
Dear @clun1,

You asked a similar question in your previous discussion: https://github.com/OpenFAST/openfast/discussions/1557.  For yaw specifically, you can invoke the simple built-in yaw override maneuvers, implement the user-defined routine `UserYawCont()` or implement a user-defined Bladed-style dynamic library.

Best regards,
